include/modules/battery.hpp

```diff
@@ -104,7 +107,11 @@ namespace modules {
     chrono::duration<double> m_interval{};
     chrono::system_clock::time_point m_lastpoll;
     thread m_subthread;
+
+    void runner();
+    void poll_events();
+    void recreate_watches();
   };
-}
+}  // namespace modules
```

Comment:
Remove useless comment please, of provide a reason for it to stay here.

Reply:
This is fine